src/libraries/System.Security.Cryptography/tests/osslplugins/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+if [ -e /usr/include/openssl/engine.h ]; then
+    echo "INFO: Building dntest ENGINE..."
+    clang -fPIC -o e_dntest.o -c e_dntest.c &&
```

Comment:
Non blocking suggestion: add a `.gitignore` to the osslplugins directory and put `*.so` in it.

.o is already taken care of by the gitignore in the project root.